tensorrt_llm/llmapi/llm_args.py

```diff
@@ -791,12 +788,6 @@ class LlmArgs(BaseModel):
     calib_config: Optional[CalibConfig] = Field(
```

Comment:
Please move `calib_config` to `TrtLlmArgs` as well

Reply:
Done, thanks.